ui/app/com/gu/recipeasy/controllers/Application.scala

```diff
@@ -22,21 +22,38 @@ class Application(override val wsClient: WSClient, override val conf: Configurat
     Ok(views.html.app("Recipeasy"))
   }
 
-  def curateRecipePage = Action { implicit request =>
+  def curateNewRecipe = Action { implicit request =>
     val newRecipe = db.getNewRecipe
     newRecipe match {
+      case Some(r) => Redirect(routes.Application.curateRecipe(r.id))
+      case None => NotFound
+    }
+  }
+
+  def curateRecipe(id: String) = Action { implicit request =>
+    val recipe = db.getRecipe(id)
+    view(recipe, editable = true)
+  }
+
+  def viewRecipe(id: String) = Action { implicit request =>
+    val recipe = db.getRecipe(id)
```

Comment:
This will fetch the recipe from the `recipe` table. Not the `curated_recipe` table. When a user edits a recipe and submits it their changes are saved to `curated_recipe` table, this means if they send the url to someone else, the recipe which gets returned will not have their edits.